devdoc/umock_c_lib_requirements_with_ids.md

```diff
@@ -174,6 +174,49 @@ Note that it is possible (and sometimes necessary) to undefine ENABLE_MOCKS:
 
 ```
 
+### ENABLE_MOCK_FILTERING
+
+`ENABLE_MOCK_FILTERING` is a define that allows specifying that only some of the functions that are mockable be actually mocked. This is usefull when it is not desired to generate mock functions for all the functions declared in a header.
+
```

Comment:
Very interesting concept, I know how it works, but I have a hard time wrapping my head around it.